content/security/for-admins/single-sign-on/configure/configure-idp.md

```diff
@@ -0,0 +1,200 @@
+3. Enter “Docker Hub SSO” or similar for the application name.
+4. Under **Supported account types**, specify who can use this application or access the app.
+5. In the **Redirect URI** section, select **Web** from the dropdown menu and paste the **Redirect URI** value from the Docker console into this field.
+6. Select **Register** to register the app.
+7. Take note of the **Client ID** from the app's overview page. You need this information to continue configuring SSO on Docker Hub.
+
+### Create client secrets for your Docker app
+
+1. Go to the Docker Hub SSO app that you created in the previous steps, then select **Certificates & secrets**.
+2. Select **+ New client secret**.
+3. Specify the description of the secret and set how long the keys can be used on Azure.
+4. Select **Add** to continue.
+5. Copy the secret **Value** field and keep it somewhere safe so you can use it to configure Docker SSO later on.
+
+### Configure API permission for Docker SSO and grant admin consent
+
+1. Go to the Docker Hub SSO app that you created in the previous steps.
+2. Navigate to the **API permission** category in your app settings.
+3. Select **Grant admin consent for YOUR TENANT NAME > Yes**.
+4. Next, you need to add additional permissions. Select **Add a permission**.
+5. Select **Delegated permissions**.
+6. Search for `Directory.Read.All`, and select this option.
+7. Then, search for `User.Read`, and select this option.
+8. Select **Add permissions**.
+
+You can verify admin consent was granted for each permission correctly by checking the **Status** column.
+
+### Assign users to the SSO app
+
+1. Navigate to your Azure AD dashboard, then select **Enterprise Applications > APP NAME**.
+2. Select **1. Assign users and groups**.
+3. Add users that will be allowed to use the app.
+
+In the Docker Console, paste the following values obtained in the previous steps to continue configuration:
+
+- **Client ID**
+- **Client Secret**
+- **Azure AD Domain**
+
+{{< /tab >}}
+{{< tab name="OneLogin" >}}
+
+1. Go to the OneLogin admin portal.
+2. Go to **Applications > Applications > Add App**
+3. Use the search input and search for “**SCIM Provisioner with SAML (SCIM v2 Core)**” and select the only result item.
+4. Enter a display name, for example “Docker Hub”.
+5. Optional: Upload Docker icons for the app and add an appropriate description for the app.
+6. Select **Save**.
+7. After saving the new app, more tabs on the left will appear. Go to the **Configuration** tab.
+8. Open a separate tab in your browser and go to **Docker Hub > Settings > Security > SSO > SAML**.
+9. Copy from the Docker console and paste the following into OneLogin:
+   - Entity ID - SAML Audience URL
+   - ACS URL - SAML Consumer URL
+   - SCIM Base URL - SCIM Base URL
+   - Custom Headers:
+     - `Content-Type: application/scim+json`
+     - `User-Agent: OneLogin SCIM`
+   - SCIM Bearer Token - SCIM Bearer Token
+   - SCIM JSON Template:
```

Comment:
this is part of SCIM configuration, not SSO

Reply:
Yeah I wasn't sure if this content should all go here or not. Would the Entity ID and ACS URL be the only thing necessary to include here?

Reply:
Removing OneLogin tab.